src/app/spaces/space.service.ts

```diff
@@ -242,7 +242,7 @@ export class SpaceService {
 
     if (!space.relationships['owned-by'] || !space.relationships['owned-by'].data) {
       space.relationalData.creator = null;
-      return;
+      return emptyObservable();
```

Comment:
@joshuawilson I also don't like error handling on lines 269-273, IMHO there should be no `console.log` and it should return `this.handleError(error);` instead of an empty observable, wdyt?

Reply:
I agree.